Doc/library/code.rst

```diff
@@ -166,7 +166,7 @@ interpreter objects as well as the following additions.
       Print an exit message when exiting.
 
 
-.. method:: InteractiveConsole.push(line)
+.. method:: InteractiveConsole.push(line, filename=None, _symbol="single")
```

Comment:
I think we should have a separate PR for re-documenting `InteractiveConsole` methods. In addition, I'm still unsure whether private parameters should be exposed or not. Personally, I would rather prefer not having them exposed but if subclasses need to override the method, I think their signature should then be compatible. Note that https://github.com/python/typeshed/blob/1c17cd429c2f91b0066547deb99c537af3e54d39/stdlib/code.pyi#L23-L37 does *not* expose this private parameter, so I think we shouldn't as well (and let this inconsistency with the signature remain; mypy wouldn't complain as typeshed also doesn't specify it.

Finally, `filename` seems to be only 3.13+ so we should also be careful with the backports (therefore I'd prefer having a separate PR for `InteractiveConsole`)